manifests/0000_21_cluster-openshift-controller-manager-operator_03_builder-deployer-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-core-operators
+  name: openshift-controller-manager-images
+data:
+  builderImage: quay.io/openshift/origin-docker-builder:v4.0
```

Comment:
@bparees is this image the right image for the builder?

Reply:
yes